http-add-on/templates/interceptor/deployment.yaml

```diff
@@ -112,6 +112,14 @@ spec:
         {{- end }}
         - name: KEDIFY_PROXY_ACCESS_LOG_TYPE
           value: "{{ .Values.kedifyProxy.accessLogType }}"
+        {{- if and .Values.interceptor.envoy .Values.interceptor.envoy.listener .Values.interceptor.envoy.listener.tcpBacklogSize }}
```

Comment:
```suggestion
        {{- if ((.Values.interceptor.envoy).listener).tcpBacklogSize }}
```

Reply:
nit: this should be the same semantics

Reply:
any nil on the way should be handled correctly (https://stackoverflow.com/a/68807258/1853417)